mods/ts/weapons/energyweapons.yaml

```diff
@@ -124,7 +124,6 @@ Proton:
 		ImpactSound: ssplash3.aud
 		ValidImpactTypes: Water
 
-# Obelisk laser
```

Comment:
Hmm, the internal weapon names don't have to match for map importing so I think this can be safely changed. Westwood's habit of enumerating weapon names isn't really a great smart idea worth copying.